sentry-sidekiq/spec/sentry/sidekiq/sentry_context_middleware_spec.rb

```diff
@@ -52,6 +52,12 @@
       expect(event.user).to eq(user)
     end
 
+    it "sets sidekiq tags to the event" do
+      execute_worker(processor, HappyWorker, tags: ["marvel", "dc"])
```

Comment:
> adding tags here isn't the same as adding tags on workers. can you add another worker with tags for this test?

Makes sense!

> also, since tags is a new feature for Sidekiq 6.0, you may need to skip the test when running against 5.x versions.

Technically, `sidekiq_options` allow any custom option, so we might have no need to check compatibility issue here?

https://github.com/mperham/sidekiq/blob/3b233c558a680b0c8d37318ad5724bb6c2fb3b91/lib/sidekiq/worker.rb?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L60-L74

Reply:
I see. looks like it's working across all versions 👍